libcontainer/rootfs_linux.go

```diff
@@ -339,6 +339,7 @@ func mountToRootfs(m *configs.Mount, rootfs, mountLabel string, enableCgroupns b
 		return label.SetFileLabel(dest, mountLabel)
 	case "tmpfs":
 		copyUp := m.Extensions&configs.EXT_COPYUP == configs.EXT_COPYUP
+		isRo := m.Flags&unix.MS_RDONLY == 1
```

Comment:
This check only works correctly because, by accident, the value of `unix.MS_RDONLY` is 1. Not that we will expect this to change, but if someone copy/pastes this code to check for the other flag, it won't work.

```suggestion
		isRo := m.Flags&unix.MS_RDONLY != 0
```

Reply:
I also personally don't think a variable is necessary in either case -- you only check the value once and to me `m.Flags&unix.MS_RDONLY != 0` is more obvious to read.

Reply:
I have removed the variable, the value will be checked only once before doing the remount.